primefaces/src/main/java/org/primefaces/model/filter/LessThanEqualsFilterConstraint.java

```diff
@@ -23,12 +23,18 @@
  */
 package org.primefaces.model.filter;
 
+import java.math.BigDecimal;
 import java.util.function.BiPredicate;
 
 public class LessThanEqualsFilterConstraint extends ComparableFilterConstraint {
 
     @Override
-    protected BiPredicate<String, String> getPredicate() {
+    protected BiPredicate<String, String> getPredicateString() {
```

Comment:
couldnt we just have one <Comparable, Comparable> Predicate instead of 2?

Reply:
I didn't try because comparables are typed and am not sure if we would be running into generic type issues. I can give it a try.

Reply:
Okay, no issues with generics. Works with both numbers and strings.